src/extended_json.ts

```diff
@@ -93,6 +93,7 @@ function deserializeValue(value: any, options: EJSONOptions = {}) {
       }
       if (in64BitRange) {
         if (options.useBigInt64) {
+          // eslint-disable-next-line no-restricted-globals
```

Comment:
One comment, one question

- Let's add a comment explaining why we're disabling the rule here.  With the custom bigint plugin, it was obvious, but now that we're relying on `no-restricted-globals`, it's no longer obvious why the linter is disabled for these lines.
- Related to the above - can we configure `no-restricted-globals` to always require an additional message / description of why we're ignoring the rule?  It would be nice if we could enforce that when lint rules are skipped, the justification is explained.

Reply:
From what I'm reading so far, the only way to do this would be to write a custom rule that checks contents of comments for the `eslint-disable` and `eslint-disable-next-line` directives and report when they don't contain what usually is an optional description.

Relevant documentation: https://eslint.org/docs/latest/use/configure/rules#comment-descriptions